File: Android/Sample/lib.style/src/main/java/com/example/mysmall/lib/style/StyleManager.java
package com.example.mysmall.lib.style;

import android.app.Application;

import net.wequick.small.Small;

/**
 * Created by galen on 16/8/9.
 */
public class StyleManager extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Small.setWebActivityTheme(R.style.AppTheme);
    }
}

File: Android/Sample/lib.style/src/main/java/com/example/mysmall/lib/style/StyleManager.kt
package com.example.mysmall.lib.style

import android.app.Application

import net.wequick.small.Small

/**
 * Created by galen on 16/8/9.
 */
class StyleManager : Application() {

    override fun onCreate() {
        super.onCreate()
        Small.setWebActivityTheme(R.style.AppTheme)
    }
}
